pandas/core/arrays/integer.py

```diff
@@ -293,6 +295,44 @@ def __array__(self, dtype=None):
         """
         return self._coerce_to_ndarray()
 
+    _HANDLED_TYPES = (np.ndarray, numbers.Number)
+
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+
+        out = kwargs.get('out', ())
+
+        for x in inputs + out:
+            if not isinstance(x, self._HANDLED_TYPES + (IntegerArray,)):
+                return NotImplemented
+
+        # for binary ops, use our custom dunder methods
+        result = ops.maybe_dispatch_ufunc_to_dunder_op(
+            self, ufunc, method, *inputs, **kwargs)
+        if result is not None:
```

Comment:
Is there a reason why you use a different sentinel value `None` rather than Python's standard `NotImplemented`?

Reply:
Not a specific reason, just that I didn't return anything in the `maybe_dispatch_ufunc_to_dunder_op`, which means the result is None (and our own dunder ops will never return None). 
But can make it more explicit.